### PR TITLE
Improve `Connect CENNZnet` flow

### LINKED ISSUE #64: Click on Connect Wallet initially cause the text to be disappeared
- Open the browser in Incognito Mode
- Click on Connect Wallet on top right corner

Screen Shot 2022-02-24 at 09 05 12

### LINKED ISSUE #69: Add Disconnect link to the Wallet Modal

### LINKED ISSUE #71: In Wallet Modal, Switch Account dropdown is being cut-off and is not scrollable
Screen Shot 2022-02-24 at 10 31 21

### PR DESCRIPTION
**Pull request by ken-centrality**
## Description

Various tweaks to improve the "Connect CENNZnet" flow plus re-styling the  `Wallet` and `WalletModal` components

Closes #64 
Closes #69  
Closes #71

#### Changes

- Move `Identicon` into its own `AccountIdenticon` component plus bringing the dynamic fetching outside of React rendering so Next.js can do its optimisation
- Use `AccountIdenticon` on Wallet button
- Use standard select for selecting different accounts for more slimline look
- Use `css` utility from `@emotion/react` package to do styling plus move all of stylings into `styles` object for legibility
- Set `balances` state to empty before fetching new balances when switching account, so we know when to display a circular progress

#### Fixes

- Pass the `modalOpen` state straight down from `Wallet` to `WalletModal` to avoid out-of-sync state issue

#### Removes

- Remove `AccountBalances` as it's now merged with `WalletModal`

#### Adds

- Re-add the `connectWallet` method to Wallet button component
- Add `Connecting` state to the `Wallet` button component

## Notes

This PR introduces a different approach to styling, [`css` prop from @emotion/react](https://emotion.sh/docs/introduction#react), which I think will help us harness `@mui/material` power effortlessly but without compromise of flexibility when we want to do custom styling.

### CODE REVIEW

File: components/Wallet.tsx [hunk 1]
```diff
@@ -1,63 +1,124 @@
...
+					{walletState === "Connected" && (
+						<span>{selectedAccount?.meta.name}</span>
+					)}
+					{walletState === "Connecting" && (
+						<span>
+							Connecting
+							<ThreeDots />
+						</span>
+					)}
+					{walletState === "NotConnected" && <span>Connect CENNZNET</span>}
```
Comment (aidan-starke) [inline components/Wallet.tsx:56]: I think our marketing team would prefer this read `CONNECT CENNZnet`

**Review by aidan-starke — APPROVED** [review]

**Comment by aidan-starke** [comment]
LGTM! One small thing is long account name doesn't show ellipsis
Screen Shot 2022-02-27 at 8 01 38 AM

**Comment by ken-centrality** [comment]
> LGTM! One small thing is long account name doesn't show ellipsis Screen Shot 2022-02-27 at 8 01 38 AM

Good spotting, and yeah I can never type the word `ellipsis` right 😕

File: components/Wallet.tsx (hunk 1 above)
Comment (ken-centrality) [reply components/Wallet.tsx]: Yip, agree, looks much better! Thanks

**Comment by aidan-starke** [comment]
I think this should fix #71 right?

**Pull request merged**